docs/README.md

```diff
@@ -32,8 +34,8 @@ The toolkit is envisioned to consist of the following libraries:
 * *Evaluator* * - Evaluates the scan results from Scanner as OK or NOT OK based on user specified approval / rejection
   ruleset.
 * *Advisor* * - Retrieves security advisories based on Analyzer results.
-* *Reporter* * - Summarizes the output from Analyzer, Scanner and Evaluator in an interactive UI that shows the identified copyrights,
-  licenses and NOT OK issues.
+* *Reporter* * - Summarizes the output from Analyzer, Scanner and Evaluator in an interactive UI that shows the
```

Comment:
Should we also remove the asterisk for the footnote form here as it's implemented now? It probably not "complete", but will it ever be 😉

Reply:
Done